packages/subgraph/schema.graphql

```diff
@@ -180,6 +180,7 @@ type ERC20VotingProposal implements Proposal @entity {
   open: Boolean!
   executed: Boolean!
   createdAt: BigInt!
+  expectedPass: Boolean
```

Comment:
perhaps `passable` might be a better naming
but not a problem, we can iterate on that.

Reply:
`executable` is maybe the one that makes more sense, but I believe it can cause confusion in the sense that is not really `executable` until after the proposal `endDate`